packages/utilities/src/customizable.tsx

```diff
@@ -41,10 +41,25 @@ export function customizable(
               const defaultProps = Customizations.getSettings(fields, scope, context.customizations);
 
               // tslint:disable-next-line:no-any
-              const componentProps = this.props as any;
+              let componentProps = this.props as any;
+
+              // If defaultProps.styles is a function, evaluate it before calling concatStyleSets
+              if (defaultProps.styles && typeof defaultProps.styles === 'function') {
+                defaultProps.styles = defaultProps.styles(componentProps);
```

Comment:
I wonder if this should be:
```tsx
defaultProps.styles = defaultProps.styles( { ...defaultProps, ...componentProps });
```
For example, to account for `theme` prop that may exist in `defaultProps`.